.github/workflows/nightly-rosetta-pax-build.yaml

```diff
@@ -123,49 +150,45 @@ jobs:
         ${{ needs.arm64.outputs.DOCKER_TAG_FINAL }}
       TARGET_IMAGE: pax
       TARGET_TAGS: |
-        type=raw,value=latest,priority=1000
-        type=raw,value=nightly-${{ needs.metadata.outputs.BUILD_DATE }},priority=900  
+        ${{ needs.test-amd64.outputs.TEST_STATUS == 'success' && 'type=raw,value=latest,priority=1000' || '' }}
```

Comment:
I'd recommend to put this logic in a 'metadata' job/jobstep, and then reference the job/jobstep output for better readability.

Reply:
This depends on test results. `metadata` is an ancestor of the test step.